examples/with_winit/src/lib.rs

```diff
@@ -180,7 +182,11 @@ fn run(
                                 stats.clear_min_and_max();
                             }
                             Some(VirtualKeyCode::M) => {
-                                aa_config_ix = (aa_config_ix + 1) % AA_CONFIGS.len();
+                                aa_config_ix = if modifiers.shift() {
```

Comment:
Personally I would keep the mod n behavior (as the reverse is less discoverable), but that's a style preference. Note though that we do have +/- and mod n for cycling through the test scenes.

Reply:
That's fair, the scene cycling logic is actually structured this way (though bound to arrow keys instead of using a modifier) so I just followed that here for consistency.